pandas/core/dtypes/concat.py

```diff
@@ -242,6 +242,72 @@ def union_categoricals(to_union, sort_categories=False, ignore_order=False):
         - sort_categories=True and Categoricals are ordered
     ValueError
         Empty list of categoricals passed
+
+    Notes
+    -----
+
+    To learn more about categories, see 'link
```

Comment:
and the starting quote should also be backtick (`` ` ``), not a single quote (like you did for the ending quote)